futaba/cogs/optional/simplewriter/words/words.py

```diff
@@ -0,0 +1,21 @@
+#
+# cogs/optional/simplewriter/words/words.py
+#
+# futaba - A Discord Mod bot for the Programming server
+# Copyright (c) 2017-2020 Jake Richardson, Ammon Smith, jackylam5
+#
+# futaba is available free of charge under the terms of the MIT
+# License. You are free to redistribute and/or modify it under those
+# terms. It is distributed in the hopes that it will be useful, but
+# WITHOUT ANY WARRANTY. See the LICENSE file for more details.
+#
+
+import os
+
+__all__ = ["core_words_list"]
+
+core_words_list = []
+core_words_list_path = os.path.join(os.path.dirname(__file__), "core_words_list.txt")
```

Comment:
You can't always expect this directory to be writeable, it might be in `/usr` or something. Just have a line in the config instead.

Reply:
This is for a read operation of [a text file in the same directory as this source file](https://github.com/JoshuaS3/futaba/blob/simplewriter/futaba/cogs/optional/simplewriter/words/core_words_list.txt). The text file will not change and is not configurable through means other than a commit due to the static nature of the [source](https://xkcd.com/simplewriter/words.js?cache=6f3c4ed1c04dc0b6c30d46e2ea451210).